src/lib.rs

```diff
@@ -47,16 +47,16 @@
 //! let mut reader = io::BufReader::new(certfile);
 //!
 //! // Load and return certificate.
-//! let certs = rustls_pemfile::certs(&mut reader).unwrap();
-//! let certs = certs.into_iter().map(rustls::Certificate).collect();
+//! let certs = rustls_pemfile::certs(&mut reader).flat_map(|x| x);
```

Comment:
Should `collect::<Result<Vec<_>, _>>().unwrap()` here. No `map()` necessary, I think?